.github/workflows/deployment.yml

```diff
@@ -33,7 +33,7 @@ jobs:
 
   Chrome:
     needs: Build
-    if: github.event_name == 'push' || needs.Build.outputs.created
+    if: github.event_name == 'workflow_dispatch' || needs.Build.outputs.created
```

Comment:
This was dead code now. We used to allow retries when new tags were pushed manually, but that is no longer possible

Reply:
That sounds fine!